### PR TITLE
Updates Outreach Committee section

### PR DESCRIPTION
**Pull request by pdp2shirts**
Updates the Outreach Committee section of the SUBGROUPS.md
Signed-off-by: Paul Pindell <ppindell@gmail.com>

### CODE REVIEW

**Comment by pdp2shirts** [comment]
@opiproject/opi-maintainers It doesn't appear that I have rights to request a review from specific people. So using this backdoor method to request a review :)

File: SUBGROUPS.md [hunk 1]
```diff
@@ -84,20 +84,18 @@
...
   * Active
 
-## Events and Outreach
+## Outreach Committee
 
 * Description & Responsibilities
-  * The charter of the subgroup is to plan, coordinate, and execute DB
-    community and other industry events as well as lead outreach, marketing,
-    and PR activities for DB
+  * The charter of the subgroup is where recruiting for additional project members, events coordination, vision discussions, and marketing related activities will take place. sometimes referred to as the project's non-technical, non-governing board activities
```
Comment (seroyer) [inline SUBGROUPS.md]: A few nits:
1. Prefer to wrap long lines
2. "sometimes" -> "Sometimes"
3. Second sentence is incomplete

**Review by seroyer — APPROVED** [review]

File: SUBGROUPS.md (hunk 1 above)
Comment (pdp2shirts) [reply SUBGROUPS.md]: @seroyer fixed 1, 2, and 3.

File: SUBGROUPS.md [hunk 2]
```diff
@@ -84,20 +84,21 @@
...
 
-## Events and Outreach
+## Outreach Committee
 
 * Description & Responsibilities
-  * The charter of the subgroup is to plan, coordinate, and execute DB
-    community and other industry events as well as lead outreach, marketing,
-    and PR activities for DB
+  * The charter of the subgroup is where recruiting for additional project members,
+  events coordination, vision discussions, and marketing related activities will
```
Comment (mestery) [inline SUBGROUPS.md]: Can you move these sentences to line up with `The` above? I'm somewhat surprised this passed the linter.

Comment (pdp2shirts) [reply SUBGROUPS.md]: done

**Review by glimchb — APPROVED** [review]
approved and merging this now

**Pull request merged**